docs-source/content/faq/oci-fss-pv.md

```diff
@@ -0,0 +1,96 @@
+update file ownership used by the standard Oracle images where
+UID `1000` and GID `1000` normally represent the `oracle` or `opc` user.
+For more information on persistent volume handling,
+see [Persistent storage]({{< relref "/userguide/managing-domains/persistent-storage/_index.md" >}}).
+{{% /notice %}}
+
+
+### Failure during domain creation with persistent volume sample
+
+The existing sample for [creation of a domain home on persistent volume](https://github.com/oracle/weblogic-kubernetes-operator/tree/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv)
+uses a Kubernetes job to create the domain. The sample uses an
+`initContainers` section to change the file ownership which will
+fail for OCI FSS created volumes used with an OKE cluster.
+
+The OCI FSS volume contains some files that are not modifiable thus
+causing the Kubernetes job to fail. The failure is see from the
+description of the Kubernetes job pod:
+```bash
+$ kubectl describe -n domain1-ns pod domain1-create-weblogic-sample-domain-job-wdkvs
+   :
+   :
+Init Containers:
+  fix-pvc-owner:
+    Container ID:  docker://7051b6abdc296c76e937246df03d157926f2f7477e63b6af3bf65f6ae1ceddee
+    Image:         container-registry.oracle.com/middleware/weblogic:12.2.1.3
+    Image ID:      docker-pullable://container-registry.oracle.com/middleware/weblogic@sha256:47dfd4fdf6b56210a6c49021b57dc2a6f2b0d3b3cfcd253af7a75ff6e7421498
+    Port:          <none>
+    Host Port:     <none>
+    Command:
+      sh
+      -c
+      chown -R 1000:1000 /shared
+    State:          Terminated
+      Reason:       Error
+      Exit Code:    1
+      Started:      Wed, 12 Feb 2020 18:28:53 +0000
+      Finished:     Wed, 12 Feb 2020 18:28:53 +0000
+    Ready:          False
+    Restart Count:  0
+    Environment:    <none>
+
+   :
+```
+
+### Updating the domain on persistent volume sample
+In the snippet of the [create-domain-job-template.yaml](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-job-template.yaml)
+below, you will see the updated `command` for the init container:
+```
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: %DOMAIN_UID%-create-weblogic-sample-domain-job
+  namespace: %NAMESPACE%
+spec:
+  template:
+    metadata:
+  :
+  :
+    spec:
+      restartPolicy: Never
+      initContainers:
+        - name: fix-pvc-owner
+          image: %WEBLOGIC_IMAGE%
+          command: ["sh", "-c", "chown -R 1000:1000 $(ls -a %DOMAIN_ROOT_DIR% | grep -v '^\\.')"]
+          volumeMounts:
+          - name: weblogic-sample-domain-storage-volume
+            mountPath: %DOMAIN_ROOT_DIR%
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
+      containers:
+        - name: create-weblogic-sample-domain-job
+          image: %WEBLOGIC_IMAGE%
+
+  :
+```
+The new `command` shown above should be made to your copy of this template file
```

Comment:
The new command shown above should be made to your copy of this template file and will result in the ownership updated only for the expected files before the WebLogic domain is created on the persistent volume. -> 
Use this new command in your copy of this template file. This will result in the ownership being updated only for the expected files before the WebLogic domain is created on the persistent volume.

Reply:
Actually, the last part of this second sentence should be, "being updated for the expected files only, before the WebLogic domain is created on the persistent volume."
(Position of only: Keep only close to that which it modifies to improve the clarity and accuracy of the sentence.)